public/assets/js/modules/cocktail.js

```diff
@@ -89,4 +89,61 @@ export function initializeCocktail() {
         const units = JSON.parse(document.getElementById('unitOptions')?.value || '[]');
         return units.map(unit => `<option value="${unit.id}">${unit.name}</option>`).join('');
     }
+
```

Comment:
It looks like this could be more organized some how and, Can you explain why we need so much js for both user and cocktail images ?

Reply:
The JS is there to handle errors for larger images.

The images selected by the user, can not go through any processing before they click on Save change or submit, therefor php handling of the errors  will not work if we want to display the error inside the forms.

The JS is there to handle those errors, and to prevent image submission on the fly.

Reply:
FIXED